newsView.test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @jest-environment jsdom
+ */
+
+ const fs = require('fs');
+ const NewsView = require('./newsView')
+ const NewsModel = require('./newsModel')
+ const NewsApi = require('./newsApi')
+
+ const newsList = [
+     {
+        "fields": {
+            "headline": "Test Headline 1",
+            "thumbline": "test_img 1",
+        },
+        "webUrl": "https://www.theguardian.com/sport/live/2022/feb/05/ireland-v-wales-six-nations-2022-live",
+    },
+    {
+        "fields": {
+            "headline": "Test Headline 2",
+            "thumbline": "test_img 2",
+        },
+        "webUrl": "https://www.theguardian.com/sport/live/2022/feb/05/ireland-v-wales-six-nations-2022-live"
+    }
+ ]
+
+ describe("NewsView class", () => {
+     it('displays news title , url and image after click load button', () =>  {
+        document.body.innerHTML = fs.readFileSync("./index.html");
+         const model = new NewsModel()
+         const view = new NewsView(model)
+         const loadButtonEl = document.querySelector('#load-news')
+            loadButtonEl.click()
+            view.displayNews()
+         expect(document.querySelectorAll('#news-list').length).toEqual(1)
```

Comment:
I'm not sure what you are trying to test here.
Are you testing that when you call displayNews the news are displayed?
OR, are you trying to test that when the button is click the news are displayed?

The second test would be an interface test whereas the first test would test the method inside the view... and actually would fail, revealing the bug pointed above ;)